javascript-bank/account.js

```diff
@@ -0,0 +1,44 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  var trueOrFalse = true;
+  if (amount > 0 && typeof amount === 'number' && Number.isInteger(amount)) {
+    this.transactions.push(new Transaction('deposit', amount));
+  } else {
+    trueOrFalse = false;
+  }
+  return trueOrFalse;
+};
+
+Account.prototype.withdraw = function (amount) {
+  var trueOrFalse = true;
+  if (amount > 0 && typeof amount === 'number' && Number.isInteger(amount)) {
+    this.transactions.push(new Transaction('withdrawal', amount));
+  } else {
+    trueOrFalse = false;
+  }
+  return trueOrFalse;
+};
+
+Account.prototype.getBalance = function () {
+  var total = 0;
+  var checkBalance = this.transactions;
+  if (checkBalance.length === 0) {
+    return 0;
+  } else {
```

Comment:
You don't need to do this check. If the length is `0` the loop will never run so `0` will be returned anyways. You can get rid of the `if else` statement